web/app/css/sidebar.css

```diff
@@ -32,6 +32,10 @@ ul.ant-menu.ant-menu-sub {
     & .update {
```

Comment:
Font feedback: looking at the sidebar we can see that the subitems are larger than the headings `Deployments` and `Pods`. Esp since the pod names are long, I think we should use a smaller font size (maybe 12px?)
![screen shot 2018-08-24 at 2 00 20 pm](https://user-images.githubusercontent.com/549258/44607657-1398f080-a7a6-11e8-8acc-41f72633f6c1.png)